setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+exclude = .git,templates,migrations
```

Comment:
This doesn't need to be in this branch but this allows us to just run `flake8 .` at the project folder and it excludes these b/c of the config. I should move this from here to master tho to keep this branch on topic.